src/client/src/routes/MainRoute.tsx

```diff
@@ -20,6 +21,7 @@ const APPLICATION_DISCONNECT = 15 * 60 * 1000
 
 const config = getEnvVars(process.env.REACT_APP_ENV!)
 const LOG_NAME = 'Application Service: '
```

Comment:
Because we no longer reference this "global" constant you can go ahead and roll this string into your logger.